sunpy/map/sources/hinode.py

```diff
@@ -36,24 +46,43 @@ class XRTMap(GenericMap):
                                   "Be_thick", "Gband", "Ti_poly"]
 
     def __init__(self, data, header, **kwargs):
-        super().__init__(data, header, **kwargs)
-
-        fw1 = header.get('EC_FW1_')
+        fw1 = header.get('EC_FW1_', '')
         if fw1.lower() not in _lower_list(self.filter_wheel1_measurements):
-            raise ValueError('Unpexpected filter wheel 1 in header.')
-
-        fw2 = header.get('EC_FW2_')
+            raise ValueError(f'Unexpected filter wheel 1 {fw1} in header.')
+        fw2 = header.get('EC_FW2_', '')
         if fw2.lower() not in _lower_list(self.filter_wheel2_measurements):
-            raise ValueError('Unpexpected filter wheel 2 in header.')
-
+            raise ValueError(f'Unexpected filter wheel 2 {fw2} in header.')
+        super().__init__(data, header, **kwargs)
         self.plot_settings['cmap'] = 'hinodexrt'
 
     @property
     def _timesys(self):
         if self.meta.get('timesys', '').upper() == 'UTC (TBR)':
             return 'UTC'
-        else:
-            return super()._timesys
+        return super()._timesys
+
+    @property
+    # @cached_property_based_on('_meta_hash')
```

Comment:
I was unsure if I should restore this behaviour here.

Reply:
I can't see why not?